google/cloud/forseti/services/inventory/base/gcp.py

```diff
@@ -85,8 +85,8 @@ def iter_crm_organization_access_levels(self, access_policy_id):
         """
 
     @abc.abstractmethod
-    def iter_crm_organization_access_policies(self, org_id):
-        """Iterate Access Policies from GCP API.
+    def fetch_crm_organization_access_policy(self, org_id):
```

Comment:
the changes in this file is not related to org policy.

Reply:
Removed